library/Garp/Spawn/Behavior/Type/Sluggable.php

```diff
@@ -68,9 +68,16 @@ public function needsPhpModelObserver() {
 	 * @return	Array	Configuration of the slug field
 	 */
 	protected function _getSlugFieldConfig() {
-		$slugFieldConfig 	= $this->_slugFieldConfig;
-
+		$slugFieldConfig = $this->_slugFieldConfig;
+		$model = $this->getModel();
 		if ($this->_baseFieldIsMultilingual()) {
+            if ($model instanceof Garp_Spawn_Model_I18n) {
```

Comment:
Ja, dat dacht ik ook. Maar het probleem is: het base model is juist multilingual, en aan de hand daarvan wordt `Garp_Spawn_Model_I18n` gegenereerd. Stom genoeg kan ik dan aan de hand van dat model niet zien dat het meertalig is.